asserts/ifacedecls.go

```diff
@@ -434,10 +441,10 @@ func compilePlugRule(interfaceName string, rule interface{}) (*PlugRule, error)
 		return nil, err
 	}
 	if rMap == nil {
-		if shortcut {
-			rMap = defaultAllowRule
-		} else {
-			rMap = denyRule
+		if shortcut == "true" {
+			rMap = defaultOutcome
+		} else { // "false"
```

Comment:
Perhaps "invert bool" is a better variable here, and makes the association with invertOutcome more clear. It's also nice that it's false by default, which means it only changes something if set.